packages/components/organisms/f-checkout/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-checkout",
   "description": "Fozzie Checkout – Fozzie Checkout Component",
-  "version": "0.78.1",
+  "version": "0.79.0",
```

Comment:
this will conflict with this PR https://github.com/justeat/fozzie-components/pull/846. I'll try to release my changes after yours.

Reply:
fixed the version in my PR